### PR TITLE
Statistics for Constant Sum question are rounded off in the csv file #3134

### LINKED ISSUE #3134: Statistics for Constant Sum question are rounded off in the csv file
In the html version of the statistics, the averages computed are in 2 decimal place. In the csv version, a rounded-down version is displayed. (e.g. 2.75 is displayed as 2 in the csv statistics)

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #3134

### CODE REVIEW

File: src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java [hunk 1]
```diff
@@ -523,7 +523,7 @@ public String getQuestionResultStatisticsCsv(
         DecimalFormat df = new DecimalFormat("#.##");
         
         for(Entry<String, Integer[]> entry : optionTotalCount.entrySet() ){
-            double average = entry.getValue()[0]/entry.getValue()[1];
+            double average = (double)entry.getValue()[0]/entry.getValue()[1];
```
Comment (damithc) [inline src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java]: Does the webpage and csv both use the same function to calculate the average? if not, they should.

**Comment by kanghj** [comment]
Some changes made. After I rewrote parts of the code, the ordering of rows in the statistics changed in FeedbackSessionsLogicTest. Should the statistics be displayed in any particular order (e.g. descending average, or by recipient/option's name)?

**Comment by damithc** [comment]
Sort by name sounds reasonable. User can sort by other columns if they want
to. If it is not sorted now, we can leave it as it is and create a separate
issue for sorting.

**Comment by kanghj** [comment]
I have created a separate issue for that.

**Comment by LowWeiLin** [comment]
Could add comments for each function to document what they do. Otherwise looks good.

**Comment by kanghj** [comment]
Will add comments for the functions before merging

**Pull request merged**